node/src/blockevents.test.ts

```diff
@@ -237,15 +255,66 @@ describe('Block Events', () => {
             assertValidBlockEventsRequestHeader(payload);
 
             const seekInfo = orderer.SeekInfo.deserializeBinary(payload.getData_asU8());
-            const start = seekInfo.getStart();
-            expect(start).toBeDefined();
-            expect(start?.getTypeCase()).toBe(orderer.SeekPosition.TypeCase.SPECIFIED);
-            expect(start?.getSpecified()?.getNumber()).toBe(Number(startBlock));
-            const stop = seekInfo.getStop();
-            expect(stop).toBeDefined();
-            expect(stop?.getTypeCase()).toBe(orderer.SeekPosition.TypeCase.SPECIFIED);
-            expect(stop?.getSpecified()).toBeDefined();
-            expect(stop?.getSpecified()?.getNumber()).toBe(Number.MAX_SAFE_INTEGER);
+
+            assertStartPositionToBeSpecified(seekInfo, Number(startBlock));
+            assertStopPosition(seekInfo);
+        });
+
+        it('Sends valid request with specified start block number and fresh checkpointer', async () => {
```

Comment:
The Java unit tests cover 4 cases (whereas here are only 3), and the Java test naming better describes the expected behaviour. It's probably worth changing tweaking these tests to match the Java ones:

1. Uses specified start block instead of unset checkpoint
2. Uses checkpoint block instead of specified start block
3. Uses default start position with unset checkpoint and no start block
4. Uses checkpoint block zero with set transaction ID instead of specified start block